modules/roulette.py

```diff
@@ -0,0 +1,163 @@
+import module
+from discord import Intents, Client, Message, TextChannel
+import globs
+import re 
+import random
+from enum import Enum
+from money import MoneyModule 
+
+MoneyModule = MoneyModule() 
+
+class RouletteModule(module.Module): 
+    def __init__(self):
+        super().__init__()    
+    async def get_res (self, msg: Message): 
+        if "!roulette" not in msg.lower(): 
+            return
+
+        class BetTypes(Enum):
+            STRAIGHT = "straight" 
+            SPLIT = "split"
+            STREET = "street"
+            CORNER = "corner"
+            SIXLINE = "sixline"
+            REDBLACK = "redblack" ## Obligatory 34 joke
+            DOZEN = "dozen"
+            COLUMN = "column"
+            ODDEVEN = "oddeven"
+            HIGHLOW = "highlow"  
+        class BetCalls(Enum): 
+            RED = "red"
+            BLACK = "black" 
+            COLUMN1 = "column1"
+            COLUMN2 =  "column2"
+            COLUMN3 = "column3"
+            DOZEN1 = "first" 
+            DOZEN2 = "second"
+            DOZEN3 = "third" 
+            EVEN = "even"
+            ODD = "odd" 
+            HIGH = "high"
+            LOW = "low"
+        class Error(Enum): 
+            ERROR = "ERROR" 
+        
+        user_id = usr_msg.author.id 
+        if user_id not in MoneyModule.data: MoneyModule.update_balance(user_id, 1000)

```

Comment:
Instead of leaving `1000` here as a magic number, create a constant variable called `STARTING_CAPITAL` and then use that here.